lib/wasi/src/runners/emscripten.rs

```diff
@@ -1,24 +1,46 @@
-#![cfg(feature = "webc_runner_rt_wasi")]
+#![cfg(feature = "webc_runner_rt_emscripten")]
 //! WebC container support for running Emscripten modules
 
 use crate::runners::WapmContainer;
 use anyhow::anyhow;
 use serde::{Deserialize, Serialize};
 use std::error::Error as StdError;
 use std::sync::Arc;
-use wasmer::{Cranelift, FunctionEnv, Instance, Module, Store};
+use wasmer::{FunctionEnv, Instance, Module, Store};
 use wasmer_emscripten::{
     generate_emscripten_env, is_emscripten_module, run_emscripten_instance, EmEnv,
     EmscriptenGlobals,
 };
 use webc::{Command, WebCMmap};
 
-#[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Hash, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
 pub struct EmscriptenRunner {
     args: Vec<String>,
+    #[serde(skip, default)]
+    store: Store,
 }
 
 impl EmscriptenRunner {
+    /// Constructs a new `EmscriptenRunner` given an `Engine`
```

Comment:
```suggestion
    /// Constructs a new `EmscriptenRunner` given an `Store`
```

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3483